.teamcity/src/subprojects/build/core/RustUtils.kt

```diff
@@ -0,0 +1,57 @@
+package subprojects.build.core
+
+import dsl.scriptFile
+import jetbrains.buildServer.configs.kotlin.BuildSteps
+import jetbrains.buildServer.configs.kotlin.BuildTypeSettings
+import jetbrains.buildServer.configs.kotlin.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.buildSteps.script
+import subprojects.Agents.OS
+
+fun BuildSteps.installRust(os: OS) {
+    if (os == OS.Windows) {
+        powerShell {
+            name = "Install Rust (rustup)"
+            scriptFile("install_rust_windows.ps1")
+        }
+    } else {
+        script {
+            name = "Install Rust (rustup)"
+            scriptFile("install_rust_unix.sh")
+        }
+    }
+}
+
+fun BuildTypeSettings.enableRustCompilation(os: OS) {
+    params {
+        param("env.KTOR_RUST_COMPILATION", "true")
+        if (os == OS.Linux) {
+            param(
+                "env.KTOR_OVERRIDE_KONAN_PROPERTIES",
+                "targetSysRoot.linux_x64=/;libGcc.linux_x64=/usr/lib/gcc/x86_64-linux-gnu/13;linker.linux_x64=/usr/bin/ld.bfd;crtFilesLocation.linux_x64=/usr/lib/x86_64-linux-gnu/"
+            )
+        }
+    }
+}
+
+fun BuildSteps.enableRustLinuxArmCompilation(os: OS) {
```

Comment:
Minor. This name feels inconsistent as we use the prefix `enable` in the function above. Maybe rename it to something like `installLinuxArmCargoTarget` (I'm not sure about the word "install" here)

Reply:
I'm for `setupRustAarch64CrossCompilation`